actions/structures.xml

```diff
@@ -39,7 +39,10 @@
                     <action id="testtaker-move" name="Move" url="/taoTestTaker/TestTaker/moveInstance" context="instance" group="none" binding="moveNode">
                         <icon id="icon-move-item"/>
                     </action>
-                    <action id="testtaker-move-to" name="Move To" url="/taoTestTaker/TestTaker/moveResource" context="resource" group="tree" binding="moveTo">
+                    <action id="testtaker-move-to" name="Move To" url="/taoTestTaker/TestTaker/moveResource" context="instance" group="tree" binding="moveTo">
+                        <icon id="icon-move-item"/>
+                    </action>
+                    <action id="class-move-to" name="Move To" url="/taoTestTaker/TestTaker/moveClass" context="class" group="tree" binding="moveTo">
```

Comment:
by binding the 2 actions to the same JS action are we creating a possible bug ?

Reply:
We did, this is why in structures I changed context to separate those. This way we avoid this issue. Yet we want to use this same js binding to preserve similar ux feeling.